paper.md

```diff
@@ -0,0 +1,78 @@
+---
+title: 'BinderHub: shareable, interactive computing environments with Kubernetes'
+tags:
+  - jupyter
+  - jupyterhub
+  - interactive computing
+  - cloud computing
+  - binder
+  - kubernetes
+  - helm
+
+authors:
+ - name: Project Jupyter
+   orcid:
+   affiliation:
+ - name: Jessica Forde
+   orcid: XXXX
+   affiliation: Project Jupyter
+ - name: Brian Granger
+   orcid: 0000-0002-5223-6168
+   affiliation: Cal Poly
+ - name: Tim Head
+   orcid: 0000-0003-0931-3698
+   affiliation: Wild Tree Tech
+ - name: Chris Holdgraf
+   orcid: 0000-0002-2391-0678
+   affiliation: University of California, Berkeley
+ - name: M Pacer
+   orcid: 0000-0002-6680-2941
+   affiliation: University of California, Berkeley
+ - name: Yuvi Panda
+   orcid: 0000-0002-8055-2870
+   affiliation: University of California, Berkeley
+ - name: Fernando Perez
+   orcid: 0000-0002-1725-9815
+   affiliation: University of California, Berkeley
+ - name: Min Ragan-Kelley
+   orcid:  0000-0002-1023-7082
+   affiliation: Simula Research Laboratory
+ - name: Carol Willing
+   orcid: 0000-0002-9817-8485
+   affiliation: Cal Poly
+date: YYYY-MM-DD
+bibliography: paper.bib
+---
+
+# Summary
+
+BinderHub enables authors to create sharable, interactive
+computational environments by specifying their environment in an online
+git repository.
+
+BinderHub runs a service that lets users put their code in a repository online, use
+the BinderHub service to build a Docker image using configuration files in that
+repository, then serve that Docker image to users via a public link. BinderHub
+will flexibly create cloud resources as users request to interact with a
+particular repository's link, and will automatically destroy these resources
+after a period of inactivity. While BinderHub is designed to fit a number of
+cloud-based workflows, it has a particular focus on improving reproducibility
+for scientific work, enabling interactive published artifacts of results, and
```

Comment:
thanks, this is an improvement, but I think it is still a bit abstract. Thinking out loud...

"it has a particular focus on improving the reproducibility of published artifacts that involve interactive scientific computations, such as those done with Jupyter Notebooks, RMarkdown, Shiny, Jupyter Widgets, etc." 

"Some usage cases that binder enables:

* Open-source projects can use binder to allow anyone in the world to try their software interactively, with zero installation
* Educators can deploy computational lessons...
* Research groups can...
"

Reply:
I like the idea of a bulleted-list...I'll give that a shot in the next push!